qa/rpc-tests/wallet.py

```diff
@@ -89,6 +89,20 @@ def run_test (self):
         assert_equal(len(node2utxos), 2)
         assert_equal(sum(int(uxto["generated"] is True) for uxto in node2utxos), 0)
 
+        # Catch an attempt to send a transaction with an absurdly high fee
+        try:
+            inputs = []
+            outputs = {}
+            utxo = node0utxos[0]
+            inputs.append({ "txid" : utxo["txid"], "vout" : utxo["vout"]})
+            outputs[self.nodes[2].getnewaddress("")] = Decimal(0.00001000)
```

Comment:
This should be ``Decimal("0.00001000")``, otherwise precision is lost in the value passed into the constructor:
```
>>> from decimal import Decimal
>>> Decimal("0.00001000")
Decimal('0.00001000')
>>> Decimal(0.00001000)
Decimal('0.000010000000000000000818030539140313095458623138256371021270751953125')
```

Reply:
Fix force pushed.